lambda/health_package/tox.ini

```diff
@@ -6,58 +6,32 @@ skipsdist = true
-    pylint components/firehose_helper.py
-    pylint components/kinesis_helper.py
-    pylint components/lambda_helper.py
-    pylint components/sqs_helper.py
-    pylint generate_metric_alarms.py
-    pylint cloudwatch_forwarder.py
-    pylint cloudwatch_alarm_forwarder.py
-    pylint cloudwatch_metric_forwarder.py
-    pylint health_monitor_lambda.py
-    pylint splunk_forwarder.py
-    pylint local_exceptions.py
-    pylint -d C0103 enrich.py
-    pylint -d W0613 lambda_handler.py
-    pylint demo/set_alarm_state.py
-    pylint demo/reset_all.py
-    flake8 tests/stubs.py
-    flake8 tests/test_generate_metric_alarms.py
-    flake8 tests/test_cloudwatch_forwarder.py
-    flake8 tests/test_health_monitor_lambda.py
-    flake8 tests/test_components_generic_helper.py
-    flake8 tests/test_components_lambda_helper.py
-    flake8 tests/test_health_event.py
-    flake8 --ignore=E501 conftest.py
-    flake8 health_event.py
-    flake8 components/generic_helper.py
-    flake8 components/firehose_helper.py
-    flake8 components/kinesis_helper.py
-    flake8 components/lambda_helper.py
-    flake8 components/sqs_helper.py
-    flake8 generate_metric_alarms.py
-    flake8 cloudwatch_forwarder.py
-    flake8 cloudwatch_alarm_forwarder.py
-    flake8 cloudwatch_metric_forwarder.py
-    flake8 health_monitor_lambda.py
-    flake8 splunk_forwarder.py
-    flake8 local_exceptions.py
-    flake8 enrich.py
-    flake8 lambda_handler.py
-    flake8 demo/set_alarm_state.py
-    flake8 demo/reset_all.py
+    flake8 --max-line-length=88 tests/stubs.py
+    flake8 --max-line-length=88 tests/test_generate_metric_alarms.py
+    flake8 --max-line-length=88 tests/test_cloudwatch_forwarder.py
+    flake8 --max-line-length=88 tests/test_health_monitor_lambda.py
+    flake8 --max-line-length=88 tests/test_components_generic_helper.py
+    flake8 --max-line-length=88 tests/test_components_lambda_helper.py
+    flake8 --max-line-length=88 tests/test_health_event.py
+    flake8 --max-line-length=88 --ignore=E501 conftest.py
+    flake8 --max-line-length=88 health_event.py
+    flake8 --max-line-length=88 components/generic_helper.py
+    flake8 --max-line-length=88 components/firehose_helper.py
+    flake8 --max-line-length=88 components/kinesis_helper.py
+    flake8 --max-line-length=88 components/lambda_helper.py
+    flake8 --max-line-length=88 components/sqs_helper.py
+    flake8 --max-line-length=88 generate_metric_alarms.py
+    flake8 --max-line-length=88 cloudwatch_forwarder.py
+    flake8 --max-line-length=88 cloudwatch_alarm_forwarder.py
+    flake8 --max-line-length=88 cloudwatch_metric_forwarder.py
+    flake8 --max-line-length=88 health_monitor_lambda.py
+    flake8 --max-line-length=88 splunk_forwarder.py
+    flake8 --max-line-length=88 local_exceptions.py
+    flake8 --max-line-length=88 enrich.py
+    flake8 --max-line-length=88 lambda_handler.py
+    flake8 --max-line-length=88 demo/set_alarm_state.py
+    flake8 --max-line-length=88 demo/reset_all.py
```

Comment:
This violates the Don't Repeat Yourself principle of programming.

1) You can configure the line length of `flake8` by adding the following two lines to the `tox.ini` file:

   ```
   [flake8]
   max-line-length = 88
   ```

   See https://flake8.pycqa.org/en/latest/user/configuration.html

2) Don't specify all the files in a folder (unless there's a reason for doing so)

   `flake8` is being run on everything in the test folder. If that's the behaviour you intend, then you should change all those lines to simply `flake8 tests` - the command will iterate over all the Python files in that folder. The same goes for the components folder and the demo folder.

   It might be that you don't want all the files in the test folder to be flake8'd in the _future_ - but you should note that because it's not clear from here that is your intention.

Reply:
The original tox.ini had this very non-DRY stuff in. Our assumption that it was there for a reason?

Reply:
@denizgenc is exactly right, however. that tox fix is correct and satisfying!